loopy/kernel/data.py

```diff
@@ -455,13 +452,13 @@ def get_arg_decl(self, ast_builder, name_suffix, shape, dtype, is_written):
 
 class ValueArg(KernelArgument):
     def __init__(self, name, dtype=None, approximately=1000, target=None,
-            is_output_only=False):
+            is_output_only=False,tags=None):
 
         KernelArgument.__init__(self, name=name,
                 dtype=dtype,
                 approximately=approximately,
                 target=target,
-                is_output_only=is_output_only)
+                is_output_only=is_output_only,tags=tags)
```

Comment:
Could you make sure this attribute is documented, and documented in the right place?